components/StyledInputGroup.js

```diff
@@ -57,7 +57,7 @@ const getBgColor = ({ error, focused, success }) => {
     return 'green.100';
   }
 
-  return 'black.50';
```

Comment:
This change is a blocker for me because it breaks the prepend background in multiple places. For example the sign up form or the color picker on the collective page:

**Before**
![image](https://user-images.githubusercontent.com/1556356/70316213-7a1b1400-181b-11ea-91a2-361191aa58e1.png)


**After**
![image](https://user-images.githubusercontent.com/1556356/70316196-7091ac00-181b-11ea-9a7e-b001ef718e4e.png)